src/components/tx/TxData.js

```diff
@@ -409,10 +461,15 @@ class TxData extends React.Component {
       )
     }
 
-    const renderGraph = (label, type) => {
+    const renderGraph = (graphIndex) => {
       return (
-        <div className="mt-3 graph-div" id={`graph-${type}`} key={`graph-${type}-${this.props.transaction.hash}`}>
-          <label className="graph-label">{label}:</label>
+        <div className="d-flex flex-column flex-lg-row align-items-start mb-3 common-div bordered-wrapper w-100">
+          <div className="mt-3 graph-div" id={`graph-neighbors`} key={`graph-${this.state.graphs[graphIndex].name}-${this.props.transaction.hash}`}>
```

Comment:
Is this id used? If so we should make it something like `graph-neighbors-${graphIndex}` (or name) to differentiate.

Reply:
It is not being used. I will remove it.